### DISCUSSION TITLE
Midi Monitor realtime messages FE and F8

### DISCUSSION THREAD

**Discussion by mtarenskeen** [RomanKubiak/ctrlr#236] [Ideas]
The continuous stream of F8 (=Midi Clock) data that is coming from my synth is making the Midi Monitor window in Ctrlr practically useless. It _could_ be a very useful tool though: By default FE (Active Sensing) and F8 (Midi Clock) should be filtered and not shown, unless I select some option that I want to see these data. That's how it works in almost every other Midi SysEx monitor tool that I have worked with. I want to be able to see CC and SysEx messages that are so important for Ctrlr, not all these FE and/or F8 data. Yes, this is a bugreport, or call it a feature request  :-)

**Comment by Godlike-Productions** [comment]
I'd actually file Active Sensing as the bug. Worst idea ever.

There is a MIDI Filter  on my MIDI monitor, but it doesn't work, and you can't crete a new filter, so perhaps implementing this would be an idea.

I'm on v 6.0.26 on Windows.

![image](https://user-images.githubusercontent.com/13851825/113067230-8c7a4780-91ee-11eb-97f6-5d529cec833c.png)

**Comment by Possemo** [comment]
Don't totally trust the Ctrlr MIDI Monitor. E.g. when you want to check timings (timestamps) it does not always show correct data. My solution is midiox as monitor using loopmidi as connection. You can then send the data from midiox to the synth. This is an absolute trustworthy method and you can filter in midiox whatever you want.
